File: patterns/java/src/refactoring_guru/patterns/bridge/example/Remotes/AdvancedRemote.java
package refactoring_guru.patterns.bridge.example.Remotes;

import refactoring_guru.patterns.bridge.example.Devices.Device;

public class AdvancedRemote extends BasicRemote {

    public AdvancedRemote(Device device) {
        super.device = device;
    }

    public void mute() {
        device.setVolume(0);
    }
}

File: patterns/cs/src/RefactoringGuru/Bridge/Example/Remotes/AdvancedRemote.cs
using System;
using RefactoringGuru.Bridge.Example.Remotes;
using RefactoringGuruBridge.Example.Devices;

namespace RefactoringGuruBridge.Example.Remotes
{
    class AdvancedRemote : BasicRemote
    {
        public AdvancedRemote(IDevice device)
        {
            base.device = device;
        }

        public void Mute()
        {
            Console.WriteLine("Remote: mute");
            device.SetVolume(0);
        }
    }
}
